components/media/VideoPlayer.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <video
+    :controls="controls"
+    :autoplay="autoplay"
+    :loop="loop"
+    :muted="muted"
+    data-qa="video"
+  >
+    <source
+      :src="url"
+      :type="type"
+      data-qa="video source"
+    >
+  </video>
+</template>
+
+<script>
+  export default {
+    name: 'VideoPlayer',
+
+    props: {
+      url: {
```

Comment:
Give all the other props have the same name as the attribute they feed, let's call this one `src`.